Create Fact “american-plum” (#63)
mn-pollinators/pollinator-facts: src/facts/american-plum.md

---
title: "American Plum "
image:
  license: CC BY 2.0
  creator: Homer Edward Price

>   url: https://www.flickr.com/photos/28340342@N08

Reviewer: Please link to the specific image, not the user profile.

  src: ../images/american-plum.jpg
  alt: American Plum blossoms
source:
  name: "Bees: An Identification and Native Plant Forage Guide"
  url: https://www.pollinatorsnativeplants.com/bees-an-identification-and-native-plant-forage-guide.html
category: habitat
---

> American plum attracts a variety of bees including bumble, cuckoo, cellophane, mining, and small sweat bees.With it’s highly fragrant flowers, the American Plum offers both pollen and nectar to visitors.

Reviewer: ```suggestion
American plum attracts a variety of bees including bumble, cuckoo, cellophane, mining, and small sweat bees. With its highly fragrant flowers, the American Plum offers both pollen and nectar to visitors.
```

image:
  license: CC BY 2.0
  creator: Homer Edward Price

>   url: www.flickr.com/photos/28340342@N08/2969269623

Reviewer: Removing the `https://` breaks the link.
```suggestion
  url: https://www.flickr.com/photos/28340342@N08/2969269623
```
